pkg/artifacts/artifacts_storage.go

```diff
@@ -59,6 +59,22 @@ func InitializeArtifactStorage(pr *v1alpha1.PipelineRun, c kubernetes.Interface,
 	return NewArtifactBucketConfigFromConfigMap(configMap)
 }
 
+// CleanupArtifactStorage will delete the PipelineRun's PVC if it exists.
+func CleanupArtifactStorage(pr *v1alpha1.PipelineRun, c kubernetes.Interface, logger *zap.SugaredLogger) error {
+	configMap, err := c.CoreV1().ConfigMaps(system.GetNamespace()).Get(v1alpha1.BucketConfigName, metav1.GetOptions{})
+	shouldCreatePVC, err := needsPVC(configMap, err, logger)
+	if err != nil {
+		return err
```

Comment:
can we add some context to these errors when we return them? e.g. something like `Couldn't determine if PVC was needed from config map: %s', err)`

Reply:
Sure thing!

Reply:
bump :D

Reply:
whoops! Updated `NeedsPVC` to give a better error.